katello/utilities/helper.rb

```diff
@@ -0,0 +1,15 @@
+module Utilities
+  module Helper
+    def last_scenario
+      File.basename(File.readlink("/etc/foreman-installer/scenarios.d/last_scenario.yaml")).split(".")[0]
+    end
+
+    def accepted_scenarios
+      ["katello", "foreman-proxy-content"]
+    end
+
+    def error_message
+      "This utility can't run on a non-katello system."
+    end
+  end
```

Comment:
This helper script will make some parts of re-using these scripts more difficult. Or rather, not simply straightforward.